.eslintrc

```diff
@@ -0,0 +1,51 @@
+{
+    "root": true,
+    "parser": "@typescript-eslint/parser",
+    "plugins": [
+      "@typescript-eslint"
+    ],
+    "extends": [
+      "eslint:recommended",
+      "plugin:@typescript-eslint/eslint-recommended",
+      "plugin:@typescript-eslint/recommended"
+    ],
+    "rules": {
+      "arrow-spacing": ["warn", { "before": true, "after": true }],
+      "brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
```

Comment:
We gotta bikeshed this. [Docs here.](https://eslint.org/docs/latest/rules/brace-style)

I prefer the default `1tbs` style. Should we take a poll on the others?